articles/reporting/summary.md

```diff
@@ -3,15 +3,15 @@ title: "Summary"
 description: "The Reports tab will load in the summary screen view with a default time range of the last hour For large senders there may be a short delay while the data is being compiled The Calendar drop down provides preset time ranges and an optional custom date time picker You..."
```

Comment:
This snippet should mirror the language below

Reply:
nice catch